### PR TITLE
Add localization for expand icon collapse/expand tap hint

### LINKED ISSUE #19909: ExpansionPanel expand icon lacks a label
We will probably need a localized value to label the icon

### LINKED ISSUE #19910: ExpansionPanel doesn't communicate expanded/collapsed state
Not sure what the exact correct behavior is, but the state of the button/expand icon is not communicated.  Possibly this could be handled with the label

### LINKED ISSUE #20739: [Accesibility]
## Steps to Reproduce

1. Enable talkback
2. Tap on an ExpansionTile
3. Reports "Double tap to activate", whereas the native equivalent reports "Double tap to expand" and "Double tap to collapse"

Image of flutter expansion tile being selected, the caption shows "Double tap to activate"

Image of native expansion tile being selected, the caption shows "Double tap to expand"

## Logs

```
[√] Flutter (Channel beta, v0.5.1, on Microsoft Windows [Version 10.0.17134.228], locale en-BE)
    • Flutter version 0.5.1 at C:\tools\flutter
    • Framework revision c7ea3ca377 (3 months ago), 2018-05-29 21:07:33 +0200
    • Engine revision 1ed25ca7b7
    • Dart version 2.0.0-dev.58.0.flutter-f981f09760

[√] Android toolchain - develop for Android devices (Android SDK 27.0.3)
    • Android SDK at C:\Android\android-sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-27, build-tools 27.0.3
    • ANDROID_HOME = C:\Android\android-sdk
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b02)
    • All Android licenses accepted.

[√] Android Studio (version 3.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 25.0.1
    • Dart plugin version 173.4700
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b02)

[√] Connected devices (2 available)
    • Redmi 3S                     • c3fadefd7d33  • android-arm64 • Android 6.0.1 (API 23)
    • Android SDK built for x86 64 • emulator-5554 • android-x64   • Android 7.0 (API 24) (emulator)

• No issues found!
```

### PR DESCRIPTION
**Pull request by jonahwilliams**
Also fixes ExpansionPanelHeader to properly merge semantics.

Fixes https://github.com/flutter/flutter/issues/20739
Fixes https://github.com/flutter/flutter/issues/19909
Fixes https://github.com/flutter/flutter/issues/19910

### CODE REVIEW

File: packages/flutter/test/material/expand_icon_test.dart [hunk 1]
```diff
@@ -37,7 +37,7 @@ void main() {
     );
 
     final IconTheme iconTheme = tester.firstWidget(find.byType(IconTheme));
-    expect(iconTheme.data.color, equals(Colors.black26));
+    expect(iconTheme.data.color, equals(Colors.black));
```
Comment (HansMuller) [inline packages/flutter/test/material/expand_icon_test.dart]: This change doesn't appear to be related to the PR or the issues the PR addresses.  In fact black doesn't seem like a disabled color..

File: packages/flutter_localizations/lib/src/l10n/material_en.arb [hunk 2]
```diff
@@ -235,5 +235,15 @@
   "reorderItemRight": "Move right",
   "@reorderItemRight": {
     "description": "The audio announcement to move an item in a Reorderable List right in the list when it is oriented horizontally."
+  },
+
+  "expandedIconTapHint": "Collapse",
+  "@expandedIconTapHint": {
+    "description": "The verb infinitive which describes what happens when an expanded ExpandIcon is activated."
```
Comment (HansMuller) [inline packages/flutter_localizations/lib/src/l10n/material_en.arb]: It might be more helpful to describe this translation as describing what happens an expanded ExpandIcon toggle button is pressed. Likewise for `@collapsedIconTapHint`.

**Review by HansMuller — COMMENTED** [review]
This generally looks fine; just want to sort out the icon theme test color change.

File: packages/flutter/test/material/expand_icon_test.dart (hunk 1 above)
Comment (jonahwilliams) [reply packages/flutter/test/material/expand_icon_test.dart]: I believe this just changed because I wrapped the app in a MaterialWidget

File: packages/flutter_localizations/lib/src/l10n/material_en.arb (hunk 2 above)
Comment (jonahwilliams) [reply packages/flutter_localizations/lib/src/l10n/material_en.arb]: This one is tricky because it is used in a very specific way to override part of Talkback's hint text. For example:

Value: expand
Talkback: "double tap to <<expand>>"

Value: to expand
Talback: "double tap to <<to expand>>"

So it needs to be phrased very carefully ... and this also depends on how Talkback translates the phrase as well.

File: packages/flutter/test/material/expand_icon_test.dart (hunk 1 above)
Comment (HansMuller) [reply packages/flutter/test/material/expand_icon_test.dart]: The iconTheme issue appears to be related to: https://github.com/flutter/flutter/issues/3188

Comment (HansMuller) [reply packages/flutter/test/material/expand_icon_test.dart]: According to the spec, the disabled color for icons should be black.withOpacity(0.38)

Comment (jonahwilliams) [reply packages/flutter/test/material/expand_icon_test.dart]: Should I wait until this is resolved or just ignore it and let it be cleaned up by the fix?

Comment (jonahwilliams) [reply packages/flutter/test/material/expand_icon_test.dart]: actually nvm, I need to make sure this is still disabled.

Comment (jonahwilliams) [reply packages/flutter/test/material/expand_icon_test.dart]: Okay, the issue was that the finder was looking for the first theme. Changed to the last theme to get the disable theme. Otherwise we get the material apps default theme

**Comment by HansMuller** [comment]
LGTM
